lib/drivers/animasDriver.js

```diff
@@ -422,6 +442,52 @@ module.exports = function (config) {
     };
   };
 
+  var requestCalibration = function() {
+    return{
+      recordType: RECORD_TYPES.CGM_CALIBRATION,
+      parser : function (payload) {
+        var dt = decodeDate(struct.unpack(payload,4,'bbbb',['monthYear','day','hour','minute']));
```

Comment:
Actually, we're always decoding it in the same format (9 times), so good catch! I'm going to move the format into the `decodeDate` function itself. (The only difference is where in the payload it starts, so I'll pass the payload and index as parameters)